rambo/vagrant_resources/vagrant/Vagrantfile.virtualbox

```diff
@@ -21,7 +21,9 @@ Vagrant.configure("2") do |config|
   end
   config.ssh.username = "vagrant"
   config.ssh.forward_agent = FORWARD_SSH
-  config.vm.hostname = VM_NAME
+  if VM_HOST != "centos-7"
```

Comment:
I fixed this so that the hostname is set again for other OSs, leaving out Cent. After this pr gets merged we should make another ticket for getting the hostname set in Cent too. I've looked around online and it definitely seems like this is supposed to work, and it may not because of some combination of a bug in Vagrant or our Vagrant box.